src/seedu/addressbook/ui/TextUi.java

```diff
@@ -165,5 +167,13 @@ private static String getIndexedListForViewing(List<String> listItems) {
     private static String getIndexedListItem(int visibleIndex, String listItem) {
         return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
     }
+    
+    private static String getPrintableString(Printable... printables){
+    	String _printables = "";
+    	for(Printable printable : printables){
+    		_printables += printable.getPrintableString() + " ";
```

Comment:
concatenating a string within a loop in this manner may not be the most efficient way of joining Strings. You may want to look into StringBuilder or other APIs to do this inside a loop